modules/minimum-required-permissions-upi-azure.adoc

```diff
@@ -0,0 +1,234 @@
+* `Microsoft.Storage/storageAccounts/write`
+====
+
+.Required permissions for creating deployment
+[%collapsible]
+====
+Microsoft.Resources/deployments/read
+Microsoft.Resources/deployments/write
+Microsoft.Resources/deployments/validate/action
+Microsoft.Resources/deployments/operationstatuses/read
+====
+
+Following are the permissions required for deleting the cluster.
+
+.Required permissions for deleting authorization resources
+[%collapsible]
+====
+* `Microsoft.Authorization/roleAssignments/delete`
+====
+
+.Required permissions for deleting compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/disks/delete`
+* `Microsoft.Compute/galleries/delete`
+* `Microsoft.Compute/galleries/images/delete`
+* `Microsoft.Compute/galleries/images/versions/delete`
+* `Microsoft.Compute/virtualMachines/delete`
+* `Microsoft.Compute/images/delete`
+====
+
+.Required permissions for deleting identity management resources
+[%collapsible]
+====
+* `Microsoft.ManagedIdentity/userAssignedIdentities/delete`
+====
+
+.Required permissions for deleting network resources
+[%collapsible]
+====
+* `Microsoft.Network/dnszones/read`
+* `Microsoft.Network/dnsZones/A/read`
+* `Microsoft.Network/dnsZones/A/delete`
+* `Microsoft.Network/dnsZones/CNAME/read`
+* `Microsoft.Network/dnsZones/CNAME/delete`
+* `Microsoft.Network/loadBalancers/delete`
+* `Microsoft.Network/networkInterfaces/delete`
+* `Microsoft.Network/networkSecurityGroups/delete`
+* `Microsoft.Network/privateDnsZones/read`
+* `Microsoft.Network/privateDnsZones/A/read`
+* `Microsoft.Network/privateDnsZones/delete`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete`
+* `Microsoft.Network/publicIPAddresses/delete`
+* `Microsoft.Network/virtualNetworks/delete`
+====
+
+.Required permissions for checking health resources
+[%collapsible]
+====
+* `Microsoft.Resourcehealth/healthevent/Activated/action`
+* `Microsoft.Resourcehealth/healthevent/Resolved/action`
+* `Microsoft.Resourcehealth/healthevent/Updated/action`
+====
+
+.Required permissions for deleting resource groups
+[%collapsible]
+====
+* `Microsoft.Resources/subscriptions/resourcegroups/delete`
+====
+
+.Required permissions for deleting storage resources
+[%collapsible]
+====
+* `Microsoft.Storage/storageAccounts/delete`
+* `Microsoft.Storage/storageAccounts/listKeys/action`
+====
+
+.Optional permissions for creating marketplace virtual machine resources
+[%collapsible]
+====
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/read`
+* `Microsoft.MarketplaceOrdering/offertypes/publishers/offers/plans/agreements/write`
+====
+
+.Optional permissions for creating compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/images/read`
+* `Microsoft.Compute/images/write`
+* `Microsoft.Compute/images/delete`
+* `Microsoft.Compute/diskEncryptionSets/read`
+* `Microsoft.Compute/diskEncryptionSets/write`
+* `Microsoft.Compute/diskEncryptionSets/delete`
+* `Microsoft.Compute/virtualMachines/deallocate/action`
```

Comment:
`Microsoft.Compute/virtualMachines/deallocate/action` is required permission for UPI. Please move it to required section of creation

Only 3 `diskEncryptionSets` related permissions would be optional. Rest are required for UPI.

Reply:
Addressed